GME/Gme/GMEApp.cpp

```diff
@@ -2972,7 +3115,62 @@ void CGMEApp::OnRunInterpreter(UINT nID) {
 	RunComponent(interpreters[nID - ID_FILE_INTERPRET1]);
 }
 
+void CGMEApp::OnRunComponentHelp(UINT nID) {
+	UINT id = (nID - ID_HELP_COMPONENTHELP_FIRST) / 2;
+	bool isEven = (nID - ID_HELP_COMPONENTHELP_FIRST) % 2 == 0;
+	CGMEEventLogger::LogGMEEvent(_T("CGMEApp::OnRunComponentHelp ") +
+		(id < plugins.GetSize() ?
+			plugins[id] : (id < plugins.GetSize() + interpreters.GetSize() ?
+				interpreters[id - plugins.GetSize()] : addons[id - plugins.GetSize() - interpreters.GetSize()])) + 
+		_T("\r\n"));
+
+	// Focus must be killed to flush ObjectInspector and Browser
+	::SetFocus(NULL);

```

Comment:
Is this necessary?

Reply:
I don't know. It was part of the code I coppied from the OnRunComponent function.
Are you just refering to the `::SetFocus(NULL);` line?

Reply:
I got rid of it. There was no change in functionality.